.ci/jobs/apm-server-benckmarks-mbp.yml

```diff
@@ -13,7 +13,8 @@
         discover-pr-forks-trust: permission
         discover-pr-origin: merge-current
         discover-tags: false
-        head-filter-regex: '(main|7\.17|8\.\d+|PR-.*)'
+        # todo: remove benchmarks_2_job
+        head-filter-regex: '(main|7\.17|8\.\d+|PR-.*|benchmarks_2_job)'
```

Comment:
This won't work, branches should exist in the upstream repo, in your case, this branch only exists in your forked repository.

You can manually trigger the job, as long as you are logged in `apm-ci`:
- https://apm-ci.elastic.co/job/apm-server/job/benchmarks/view/change-requests/job/PR-8180/